libs/dh/market-participant/actors/feature-actors/src/lib/drawer/dh-actor-drawer.component.ts

```diff
@@ -82,56 +78,52 @@ import { DhBalanceResponsibleRelationTabComponent } from './balance-responsible-
   ],
   viewProviders: [DhActorAuditLogService],
   imports: [
-    TranslocoDirective,
     TranslocoPipe,
-    RxPush,
-    WATT_DRAWER,
+    TranslocoDirective,
+
     WATT_TABS,
     WATT_CARD,
+    WATT_DRAWER,
     WattDescriptionListComponent,
     WattDescriptionListItemComponent,
     WattButtonComponent,
     WattSpinnerComponent,
+
     VaterStackComponent,
-    DhFeatureFlagDirective,
+
     DhEmDashFallbackPipe,
-    DhPermissionRequiredDirective,
-    DhActorsEditActorModalComponent,
-    DhActorStatusBadgeComponent,
+    DhFeatureFlagDirective,
     DhB2bAccessTabComponent,
     DhDelegationTabComponent,
     DhCanDelegateForDirective,
-    DhBalanceResponsibleRelationTabComponent,
     DhActorAuditLogTabComponent,
+    DhActorStatusBadgeComponent,
+    DhPermissionRequiredDirective,
+    DhActorsEditActorModalComponent,
+    DhBalanceResponsibleRelationTabComponent,
   ],
 })
 export class DhActorDrawerComponent {
-  private readonly apollo = inject(Apollo);
   private readonly permissionService = inject(PermissionService);
-  private readonly closed$ = new Subject<void>();
+  private readonly destoryRef = inject(DestroyRef);
```

Comment:
nitpick: should be `destroyRef`, not `destoryRef`

Reply:
Fixed